### ISSUE TITLE
Break select dropdown into two?

### ISSUE THREAD

**Issue by laryn** [backdrop-contrib/copy_blocks#8]
Right now selecting the Layout and Region to copy are done in a single dropdown:
![Screen Shot 2020-01-29 at 11 41 37 AM](https://user-images.githubusercontent.com/871421/73382009-61176800-428c-11ea-9aaa-77bf7453ffc4.jpg)

It might be easier for the end user if this is broken into two fields: first select the Layout you want to copy from, then select the block/region from that Layout.

**Comment by docwilmot** [comment]
This is a good idea. Will look into it ASAP.

**Comment by docwilmot** [comment]
Hi @laryn @stpaultim can you check the latest code please? I've changed to an AJAX select.

**Comment by laryn** [comment]
@docwilmot this looks a lot cleaner, I like it!

![Screenshot](https://user-images.githubusercontent.com/871421/90303938-35e71580-de78-11ea-95ea-635a9a8f0cb8.png)

[Code changes](https://github.com/backdrop-contrib/copy_blocks/compare/3afb6089f0c4a82796c14940005009f0c740337c..18c71a4a436ef1ca85ff5c04ff9b53c9e9e7951e)

**Comment by stpaultim** [comment]
I did a quick preview and this is a huge step forward. I hope to get back and play with it some more. Thanks for doing this, it's much better than I was able to do. I look forward to learning from your code.

**Comment by stpaultim** [comment]
@docwilmot - Thanks so much for that PR. I would not have been able to make that change myself and believe that this is a big improvement on it's own. 

However, I wanted to see if I could make a few additional aesthetic improvements on top of what you already did. Please, feel free to pick and choose what you accept or ignore the entire thing. This was a learning experience for me. 

Included in this PR are a couple of changes:

1) I changed the block names. The previous block names felt a bit "machiney." Why not:

Block copy block = Copy block 
Region copy block = Copy region

![Home_page__home____Kittens](https://user-images.githubusercontent.com/3144571/90326072-81b5bf80-df49-11ea-9c35-4d8e1004c32a.png)

or 

Block copy block = Copy blocks block
Region copy block = Copy blocks region

I really like starting with "copy" because then they end up next to each other on the list of blocks. I prefer the first set for simplicity, but the second set includes the module name in each block title. 

2) I changed the labels on the block configuration fields to be more natural / explanatory (at the expense of being a bit wordy). 

Select layout = Copy block from this layout
Select Block = Copy this block
Select layout = Copy region from this layout
Select region = Copy this region

![Home_page__home____Kittens](https://user-images.githubusercontent.com/3144571/90326189-ea516c00-df4a-11ea-8181-fac44e15bd6a.png)

3) I added a little styling to the list of blocks to make them even easier to read. I might have broken some design rules or standards in doing so, but I thought I'd give it a try. Open to other suggestions. 

![Home_page__home____Kittens](https://user-images.githubusercontent.com/3144571/90326042-fb00e280-df48-11ea-9438-a39ed01d352f.png)

**Comment by ghost** [comment]
@stpaultim I'm not the maintainer here (so maybe I should shut up :slightly_smiling_face: ) but I feel your suggestions would be better in a new issue.

It looks like this one has already been fixed anyway. I've tried the latest dev code and it's much better than the latest release. So I'd say move @stpaultim's suggestions to a new issue and close this one as fixed.

**Comment by stpaultim** [comment]
@BWPanda - No problem. Your suggestion makes sense. I'll do that. I certainly don't want to hold up the release of existing improvements.

**Comment by stpaultim** [comment]
Done, I created an new issue. https://github.com/backdrop-contrib/copy_blocks/issues/15
